the-collab-lab/tcl-79-smart-shopping-list
# Add item to firestore

## Description

New feature: Add new item to list by saving it to Firestore.
Used a single useState for the form data.

## Related Issue
Closes #5



## Acceptance Criteria
UI-related tasks:
	- “Soon”, corresponding to 7 days
	- “Kind of soon”, corresponding to 14 days
	- “Not soon”, corresponding to 30 days
Data-related tasks:

## Type of Changes
`feature`

## Updates

### Before

![image](https://github.com/user-attachments/assets/09e027c2-5bec-4dd3-a311-62a062ec2e9e)
![image (1)](https://github.com/user-attachments/assets/a3cb0a87-78c2-4e34-b2fe-4a9334bed8a6)

### After
![Screenshot 2024-08-21 at 15 34 30](https://github.com/user-attachments/assets/2c8fb476-a506-45f8-ac96-736d9279703b)
![Screenshot 2024-08-21 at 15 34 52](https://github.com/user-attachments/assets/d4064b75-1b14-423f-8235-fc26e9c6cfbb)



## Testing Steps / QA Criteria

- Do a `git pull` and `git checkout  mm-jo-rendered-list`.
- Open the homepage by running `npm start`.
- Navigate to `ManageList`. 
- Complete the form to add a new item.
- Submit using enter or clicking Add Item button.
- You should see a success message.
- Navigate to List to see the added item.
